docs/lib/project-setup/fragments/android/project-setup/20_createProject.md

```diff
@@ -0,0 +1,22 @@
+**Open Android Studio.**  Select **+ Start a new Android Studio project.**
```

Comment:
Without step numbers, a reader switching back and forth will lose their place.

Reply:
Not sure what you're suggesting here.  This:
```
### Step 1: Create a new Project
```
is included in file:
```
docs/lib/project-setup/fragments/native_common/project-setup/common.md
```

Reply:
Not going to update this.